cmd/kube-controller-manager/app/options/options.go

```diff
@@ -437,7 +437,6 @@ func (s *KubeControllerManagerOptions) Validate(allControllers []string, disable
 	errs = append(errs, s.Authentication.Validate()...)
 	errs = append(errs, s.Authorization.Validate()...)
 	errs = append(errs, s.Metrics.Validate()...)
-	errs = append(errs, utilversion.ValidateKubeEffectiveVersion(s.ComponentGlobalsRegistry.EffectiveVersionFor(utilversion.DefaultKubeComponent)))
```

Comment:
it still seems like we need the validation here, to make sure we're in an acceptable range, even if the range widens to allow more versions